code/iaas/model/src/main/java/io/cattle/platform/core/constants/ClusterConstants.java

```diff
@@ -0,0 +1,10 @@
+package io.cattle.platform.core.constants;
+
+public class ClusterConstants {
+
+    public static final String KIND = "cluster";
+
+    public static final String ADD_REMOVE_HOST_PARAM = "host";
```

Comment:
To be consistent with the current API conventions, the value should be "hostId".  Changing this of course has a trickling effect on the json files and all the tests.